tests/rkt_env_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+)
+
+func init() {
+}
+
+var simpleTests = []struct {
+	rktCmd string
+	expect string
+}{
+	{
+		`../bin/rkt --debug --insecure-skip-verify run ./rkt-inspect-print-var-from-manifest.aci`,
```

Comment:
Would it be sensible to put the `../bin/rkt --debug --insecure-skip-verify` into a function? I think it is going to be repeated to death by all other tests.

Something like:
```go
func rktCmd(string rest) string {
	return fmt.Sprintf("../bin/rkt --debug --insecure-skip-verify %s", rest)
}
```